src/main/java/Main.java

```diff
@@ -0,0 +1,172 @@
+import spark.Spark;
+import spark.ModelAndView;
+import spark.Session;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.*;
```

Comment:
While we haven't discussed this importing * is generally bad because you get a ton of extra stuff with it.  You can change your IntelliJ settings to make it have like 20 imports before replacing with *